### PR TITLE
IKEA Kallax 77x42cm

### PR DESCRIPTION
**Pull request by pablobacho**
New part

### CODE REVIEW

**Comment by pablobacho** [comment]
Hold on. I forgot to use fusion to make it a single item.

**Comment by pablobacho** [comment]
Now I think it is right.

File: Furniture/Shelf/IKEA_Kallax_77x42cm.step [hunk 1]
```diff
@@ -0,0 +1,1931 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('FreeCAD Model'),'2;1');
+FILE_NAME(
+'/home/pablo/Workspace/FreeCAD/FreeCAD-library/Furniture/Shelf/IKEA_
```
Comment (luzpaz) [inline Furniture/Shelf/IKEA_Kallax_77x42cm.step]: Is it possible to remove the path in  `FILE_NAME()`

Comment (pablobacho) [reply Furniture/Shelf/IKEA_Kallax_77x42cm.step]: Done.

**Comment by Obijuan** [comment]
Thanks!

**Pull request merged**